content/doc/pipeline/index.adoc

```diff
@@ -1,981 +1,437 @@
 ---
 layout: simplepage
-title: "Pipeline"
+title: "Getting Started with Pipelines"
 ---
 
-:toc:
+== Audience and Purpose
 
-# Jenkins Pipeline
+This document is intended for novice users of the Jenkins pipeline feature. The document explains what a pipeline is, why that matters, and how to create the different kinds of pipelines.
 
-Jenkins is a powerful, open source automation tool with a powerful plugin architecture that helps development teams automate their software lifecycle. Jenkins is used to power many industry-leading companies' software development pipelines.
+== Why Pipeline?
 
-Jenkins Pipeline is a powerful, first-class feature for managing complex, multi-step pipelines. Jenkins Pipeline, a set of open source plugins and integrations, brings the power of Jenkins and the plugin ecosystem into a scriptable Domain Specific Language (DSL). Best of all, like Jenkins core, Pipeline is extensible by third-party developers, supporting custom extensions to the Pipeline DSL and various options for plugin integration.
+While standard Jenkins “freestyle” jobs support simple continuous integration by allowing you to define sequential tasks in an application lifecycle, they do not create a persistent record of execution, enable one script to address all the steps in a complex workflow, or confer the other advantages of pipelines.
 
-Jenkins Pipeline was known as Jenkins Workflow prior to version 1.13.
+In contrast to freestyle jobs, pipelines enable you to define the whole application lifecycle.  Pipeline functionality helps Jenkins to support continuous delivery (CD). The Pipeline plugin was built with community requirements for a flexible, extensible, and script-based CD workflow capability in mind. 
```

Comment:
I'll modify the statement

On Wed, Mar 16, 2016 at 11:33 AM, R. Tyler Croy <notifications@github.com>
wrote:

> In content/doc/pipeline/index.adoc
> <https://github.com/jenkins-infra/jenkins.io/pull/136#discussion_r56355576>
> :
>
> >
> > -Jenkins Pipeline was known as Jenkins Workflow prior to version 1.13.
> > +In contrast to freestyle jobs, pipelines enable you to define the whole application lifecycle.  Pipeline functionality helps Jenkins to support continuous delivery (CD). The Pipeline plugin was built with community requirements for a flexible, extensible, and script-based CD workflow capability in mind.
>
> "The pipeline plugin was built with community requirements for a flexible"
>
> This isn't a very useful statement, this is the Jenkins site, everything
> here is built by the community.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkins-infra/jenkins.io/pull/136/files/104e1a44735350a4043484b827561fe7641862c8#r56355576>
>